core/block_flyout_inflater.ts

```diff
@@ -101,6 +101,14 @@ export class BlockFlyoutInflater implements IFlyoutInflater {
       ) {
         blockDefinition['disabledReasons'] = [MANUALLY_DISABLED];
       }
+      // These fields used to be allowed and may still be present, but should
+      // now be ignored.
```

Comment:
Is there anywhere else in the codebase that may use BlockInfo? I'm curious if there's a better location for this (like deserialization) that may be more comprehensive, or is this really the only place where it does and will ever matter?

Reply:
Ah I didn't actually fully have that context, then. Could you maybe update the comment here to perhaps explain that exact thing? Maybe something like:

```ts
      // These fields used to be allowed and may still be present, but should
      // now be ignored for the flyout since everything should always be laid
      // out linearly.
```

Reply:
Done.